### PR TITLE
update to latest azure spring boot release

### PR DESCRIPTION
**Pull request by sophiaso**

### CODE REVIEW

**Comment by sophiaso** [comment]
Issue related: https://github.com/Microsoft/spring-data-cosmosdb/issues/212

**Comment by yuwzho** [comment]
Please hold on this PR until Jenkins World demo finished by Step. 19th US time.

File: src/main/java/com/microsoft/azure/sample/controller/TodoListController.java [hunk 1]
```diff
@@ -54,9 +51,13 @@ public TodoListController() {
     @RequestMapping(value = "/api/todolist", method = RequestMethod.GET, produces = {MediaType.APPLICATION_JSON_VALUE})
     public ResponseEntity<?> getAllTodoItems() {
         try {
-            return new ResponseEntity<List<TodoItem>>(todoItemRepository.findAll(), HttpStatus.OK);
+            final Iterable<TodoItem> itemIterable = todoItemRepository.findAll();
+            final List<TodoItem> items = new ArrayList<>();
+            itemIterable.forEach(todoItem -> items.add(todoItem));
```
Comment (yuwzho) [inline src/main/java/com/microsoft/azure/sample/controller/TodoListController.java]: Just curious why not returning the Iterable directly?

Comment (sophiaso) [reply src/main/java/com/microsoft/azure/sample/controller/TodoListController.java]: This is an update according to API change, previously `findAll` returns `List`, while new version returns `Iterable`, just updated to remove the convert and the generics `List<TodoItem>` type.

**Comment by Matthew-Dong** [comment]
Seems the Jenkins World is closed and Spring Data updated to fix some issues, I'll close and merge it.

**Pull request merged**